README.adoc

```diff
@@ -0,0 +1,279 @@
+== The Project
+
+WARNING: This template is in progress. The README is out of date, and at any given time it might not even be working. This
```

Comment:
You should really edit this if you're going to add it.  It talks about the wrong project.